resources/views/student-council/voting/view_question.blade.php

```diff
@@ -0,0 +1,53 @@
+@extends('layouts.app')
+
+@section('title')
+<a href="{{route('voting')}}" class="breadcrumb">@lang('voting.assembly')</a>
+<a href="{{route('voting.view_sitting', $question->sitting()->id)}}" class="breadcrumb">{{ $question->sitting()->title }}</a>
+<a href="#!" class="breadcrumb">{{ $question->title }}</a>
+@endsection
+@section('admin_module') active @endsection
+
+@section('content')
+
+<div class="row">
+    <div class="col s12">
+        <div class="card">
+            <div class="card-content">
+                <span class="card-title">{{ $question->title }}</span>
+                @cannot('view_results', $question)
+                <p>@lang('voting.only_after_closure')</p>
+                @endcan
```

Comment:
<blockquote> instead of <p>

Reply:
Here I don't quite understand; have I done it wrong with the @lang directive?